MaxText/inference_microbenchmark.py

```diff
@@ -261,7 +279,7 @@ def main(config):
         benchmark_loop_iters
       )
 
-      benchmark_results["Prefill_Insert"][prefill_length], decode_state = prefill_insert_benchmark(
+      benchmark_results["prefill-insert"][prefill_length], decode_state = prefill_insert_benchmark(
```

Comment:
Maybe it's cleaner if we do something like this:

```
prefill_insert_time, decode_state = prefill_insert_benchmark(
benchmark_results["insert"][prefill_length]["time_in_ms"] = prefill_insert_time - benchmark_results["prefill"][prefill_length]["time_in_ms"]
```

Reply:
Done.